### DISCUSSION TITLE
Multi schemas compatibility with AzureFunctions package

### DISCUSSION THREAD

**Discussion by dayervi** [ChilliCream/hotchocolate#5940] [Q&A]
Hello,

I'm thinking of migrating a .net projet from graphql-dotnet to hot chocolate. My main requirements are to be able to run my server on azure functions and to expose multiple distinct schemas (with shared stuffs) through multiples routes.

Most documentation and examples concern web api and it's possible with multiple graphql server services and endpoints but It seems not the same with the integration package for azure functions. The `IGraphQLRequestExecutor` seems not build to handle multiple apiRoute.

Am I missing something or is there a way to configure multiple schemas and http triggers?

thanks

**Comment by venkicompany** [comment]
I am also stuck with this. I need to host multiple GraphQL schemas in single Azure function app.  Any direction on how to achieve this?

**Reply by marcsstevenson** [reply]
I'm stuck here also. There is this project which bridges support but that is very out of date now: [https://github.com/marcomedrano/HotChocolate.AzureFunctions](https://github.com/marcomedrano/HotChocolate.AzureFunctions)

**Reply by dayervi** [reply]
I switch to an app service a while ago personally.

**Reply by marcsstevenson** [reply]
Have decided to deploy a second Azure Function instead as they're very cheap to run.
